packages/nimble-components/src/table/specs/table-column-specs/table-column-formatted-text.md

```diff
@@ -165,22 +165,17 @@ Nimble will introduce `nimble-table-column-number-text` which formats a numeric
 -   `format` - a string which controls how the number is formatted for display. It can take one of the following values:
     -   `undefined` - use a default formatter that will display integers with no trailing zeros, limits to 6 digits, and exponential notation is used for numbers that are large (\`>= 1e6\`) or small (\`< 1e-3\`) in magnitude. Will be displayed left-aligned by default (since numbers will display an inconsistent number of fractional digits).
-    -   `'round-to-integer'` - format all values as integers, rounding to nearest if the value isn't an integer and never displaying exponential notation. Will be displayed right-aligned by default.
-    -   `'decimal'` - format all values as decimal values (e.g. 123.45), always displaying `decimal-digits` digits after the separator and never displaying exponential notation. Will be displayed right-aligned by default.
-    -   `'custom'` - use [`Intl.NumberFormat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat) configured with values specified in other attributes. Will be displayed left-aligned by default (since Nimble can't know if the provided configuration will produce consistent decimal separators).
-    -   This could be extended to other pre-configured formats in future. Their configuration attributes would similarly be prefixed with the name of the format mode.
-    -   **Note:** all of the above will be implemented using a `Intl.NumberFormat` formatter. For all modes besides `custom` Nimble will configure the formatter with defaults to match the [visual design spec](https://github.com/ni/nimble/issues/887). The exception is that we will set `useGrouping: true` to achieve `1,000` rather than `1000` because this styles the values in a way that is more human readable.
+    -   `'decimal'` - format all values as decimal values (e.g. 123.45), always displaying `decimal-digits` digits after the separator and never displaying exponential notation. If required, values will be rounded to reach the specified number of decimial digits. Configuring `decimal-digits` to `0` will round the value to the nearest integer and display it with no decimal places. Will be displayed right-aligned by default.
+    -   This could be extended to other pre-configured formats in future. Their configuration attributes would be prefixed with the name of the format mode.
+    -   **Note:** all of the above will be implemented using a `Intl.NumberFormat` formatter. Nimble will configure the formatter with defaults to match the [visual design spec](https://github.com/ni/nimble/issues/887). The exception is that we will set `useGrouping: true` to achieve `1,000` rather than `1000` because this styles the values in a way that is more human readable.
 -   `decimal-digits` - when format is `decimal`, a number that controls how many digits are shown to the right of the decimal separator. Defaults to 2.
```

Comment:
```suggestion
-   `decimal-digits` - when format is `decimal`, a number that controls how many digits are shown to the right of the decimal separator. Defaults to 2 if unspecified.
```

Reply:
Also maybe we should say that decimal-digits is only used by the decimal formatter?

Reply:
I've applied your suggestion.

> Also maybe we should say that decimal-digits is only used by the decimal formatter?

The docs already say "when format is `decimal`, ....". Do you think we need to be more explicit than that?

Reply:
Could be explicit about the behavior in other formats. Maybe "For other formats specifying the decimal digits results in a invalidDecimalDigits column validation error"

Reply:
Is that the behavior we want? That's not the behavior that's currently implemented. Right now, if the format is anything other than `decimal`, then we don't perform validation on the value of `decimalDigits`. I did that to be consistent with the date-text column, which does validation by trying to create a formatter. As a result, it only validates the values that will be used as part of that formatter.

Reply:
Ah, that was me misinterpreting the validation code when scanning quickly. "For other formats the decimal digits is ignored"